sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryDateTimeRangeTests.cs

```diff
@@ -46,7 +47,17 @@ public void ToStringStartEndTests()
         [TestCase("2021-01-02T03:04:05.0000000Z/2021-05-04T03:02:01.0000000Z")]
         public void CanRoundtrip(string range)
         {
-            Assert.AreEqual(range, DateTimeRange.Parse(range).ToString());
+            Assert.AreEqual(range, MonitorQueryDateTimeRange.Parse(range).ToIsoString());
+        }
+
+        [RunOnlyOnPlatforms(Windows = true, Reason = "Default formatting differs between platforms.")]
```

Comment:
I didn't even know we had this attribute.  Should we also add some tests specific to other platforms to make sure we don't have any embarrassing surprises?

Reply:
Considering how simple the ToString logic is and that the differences in the behavior are coming from the BCL I'm inclined to leave the test as is.